ui/src/saros/stf/server/bot/jquery/JQueryHelper.java

```diff
@@ -0,0 +1,100 @@
+package saros.stf.server.bot.jquery;
+
+import de.fu_berlin.inf.ag_se.browser.extensions.IJQueryBrowser;
+import de.fu_berlin.inf.ag_se.browser.html.ISelector;
+import de.fu_berlin.inf.ag_se.browser.html.ISelector.NameSelector;
+import java.util.ArrayList;
+import java.util.List;
+
+ public class JQueryHelper {
```

Comment:
Please add javadoc to the class and its methods.

Reply:
Done